addons/l10n_latam_check/wizards/l10n_latam_payment_mass_transfer.py

```diff
@@ -53,7 +53,7 @@ def default_get(self, fields_list):
                 raise UserError(_("The register payment wizard should only be called on account.payment records."))
             checks = self.env['l10n_latam.check'].browse(self._context.get('active_ids', []))
             if checks.filtered(lambda x: x.payment_method_line_id.code != 'new_third_party_checks'):
-                raise 'You have select some payments that are not checks. Please call this action from the Third Party Checks menu'
+                raise UserError(_('You have select some payments that are not checks. Please call this action from the Third Party Checks menu'))
```

Comment:
```suggestion
                raise UserError(_('You have selected some payments that are not checks. Please call this action from the Third Party Checks menu'))
```

Reply:
@maq-adhoc don't we need the translation?

I modified the string because it was incorrect, even the one we had. Since we’re already modifying this file, we could import the string, but I think we should then add the proper translation.

Thanks!